### PR TITLE
a few package list changes

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: fedora-coreos-base.yaml [hunk 1]
```diff
@@ -77,7 +77,7 @@ packages:
  - podman skopeo runc
  # Networking
  - bridge-utils nfs-utils biosdevname iptables-services
- - NetworkManager dnsmasq
+ - NetworkManager dnsmasq hostname
  # Storage
  - container-storage-setup cloud-utils-growpart
```
Comment (dustymabe) [inline fedora-coreos-base.yaml]: while we're in here I'd like to ask.. do we need container-storage-setup ?

File: fedora-coreos-base.yaml [hunk 2]
```diff
@@ -77,7 +77,7 @@ packages:
  - podman skopeo runc
```
Comment (dustymabe) [inline fedora-coreos-base.yaml]: i wonder if we should start pulling in moby-engine ?

File: fedora-coreos-base.yaml [hunk 3]
```diff
@@ -90,6 +90,8 @@ packages:
  - authconfig sssd shadow-utils
  - logrotate
  # Used by admins interactively
- - sudo coreutils less tar xz gzip bzip2 rsync tmux
+ - sudo coreutils less tar xz gzip bzip2 tmux
  - nmap-ncat net-tools bind-utils
  - bash-completion
+ # Moving files around
+ - rsync fuse-sshfs
```
Comment (dustymabe) [inline fedora-coreos-base.yaml]: i've always wanted fuse-sshfs in so I could easily use vagrant-sshfs

File: fedora-coreos-base.yaml (hunk 1 above)
Comment (cgwalters) [reply fedora-coreos-base.yaml]: https://pagure.io/atomic-wg/issue/384

**Pull request merged**